uint8 neck_servo_pan
uint8 neck_servo_tilt
uint8 eye_servo_pan
uint8 eye_servo_tilt


